benchmark/benchmain/main.go

```diff
@@ -461,32 +491,45 @@ func streamCaller(stream testgrpc.BenchmarkService_StreamingCallClient, reqSize,
 }
 
 func runBenchmark(caller rpcCallFunc, start startFunc, stop stopFunc, bf stats.Features, s *stats.Stats, mode string) {
-	// Warm up connection.
-	for i := 0; i < warmupCallCount; i++ {
-		caller(0)
+	// if SleepBetweenRPCs > 0 we skip the warmup because otherwise
+	// we are going to send a set of simultaneous requests on every connection,
+	// which is something we are trying to avoid when using SleepBetweenRPCs param.
```

Comment:
Nit: wrap comments to 80 characters please.

Reply:
done.